app/src/components/combobox/styles.tsx

```diff
@@ -9,6 +9,8 @@ export const comboBoxCSS = css`
     --combobox-vertical-padding: 10px;
     --combobox-horizontal-padding: var(--ac-global-dimension-static-size-200);
   }
+  display: flex;
+  flex-direction: column;
```

Comment:
Necessary to stack combobox desc and error texts
![image](https://github.com/user-attachments/assets/c69f0e96-9f6e-4055-a8ae-d259864af6e5)

Reply:
In the other components the error replaces the description. I would make it so